product_manufacturer_add_supplierinfo/__init__.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 # © 2015-17 Eficent Business and IT Consulting Services S.L.
 # - Jordi Ballester Alomar
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```

Comment:
Please reserve this for authors. Contributors who migrate can go in the contributors section in the readme.